### DISCUSSION TITLE
How can you do this

### DISCUSSION THREAD

**Discussion by andrey170183** [piitaya/lovelace-mushroom#695] [Show and tell]
Hello
Is it possible to implement additional buttons such as the overlay of cards (see attached picture) or additional informative icons like "badge_icon" type for the "mushroom-template-card" to the type mushroom-vacuum-card type?

![1](https://user-images.githubusercontent.com/42129962/185793039-b156f9ba-dc82-431c-b222-017740fec39f.jpg)
.

**Comment by piitaya** [comment]
It should be possible with card mod and vertical stack in card. But there is no official way to do this.
